src/ol3-layerswitcher.js

```diff
@@ -190,7 +190,7 @@ ol.control.LayerSwitcher.prototype.renderLayer_ = function(lyr, idx) {
         li.appendChild(input);
 
         label.htmlFor = lyrId;
-        label.innerHTML = lyrTitle;
+        label.innerHTML = '&nbsp' + lyrTitle;
```

Comment:
@walkermatt If you take a look at the before/after images in the PR it adds the spacing between the <li> icon and text. This could also be accomplished with css but I felt this was a simpler solution.